phockup/phockup.py

```diff
@@ -0,0 +1,412 @@
+#!/usr/bin/env python3
+import argparse
+import importlib
+import logging
+import logging.handlers
+import os
+import re
+import sys
+
+from phockup.internal.date import Date
+from phockup.internal.dependency import check_dependencies
+from phockup.internal.phockup import Phockup
+from importlib.metadata import version
+
+PROGRAM_DESCRIPTION = """\
+Media sorting tool to organize photos and videos from your camera in folders by year, \
+month and day.
+The software will collect all files from the input directory and copy them to the output
+directory without changing the files content. It will only rename the files and  place
+them in the proper directory for year, month and day.
+"""
+
+DEFAULT_DIR_FORMAT = ['%Y', '%m', '%d']
+
+logger = logging.getLogger('phockup')
+
+
+def parse_args(args=sys.argv[1:]):
+    parser = argparse.ArgumentParser(
+        description=PROGRAM_DESCRIPTION,
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    try:
+        parser.version = f"v{version('phockup')}"
+    except importlib.metadata.PackageNotFoundError:
+        parser.version = "unknown-not-installed"
```

Comment:
If phockup is runned by hand, the software can't access to the version. I need to check is something is possible to mitigate this.

Reply:
mitigation work on python 3.11+